rum/rum-android-mobile.mdx

```diff
@@ -0,0 +1,294 @@
+      | `disableSlowRenderingDetection()` | Disables slow or frozen frame render detection which is enabled by default |
+      | `setSlowRenderingDetectionPollInterval(Duration)` | Sets default polling for slow or frozen render detection. Default detection interval is 1000 milliseconds |
+      </Accordion>
+  </Accordion>
+
+  <Accordion title="Step 3: HTTP Instrumentation Config [Optional]">
+    Integrate with OkHTTP3 to monitor HTTP events across user devices.
+
+    ```java OkHTTP3
+    private Call.Factory buildOkHttpClient(Middleware middleware) {
+      return middleware.createRumOkHttpCallFactory(new OkHttpClient());
+    }
+    ```
+
+  </Accordion>
+
+</AccordionGroup>
+
+# Instrument Your Android Application
+
+### Set Global Attributes
+
+Global attributes serve as metadata or contextual information that can be attached to telemetry, traces, or logs collected by the instrumentation framework or SDK. They are key-value pairs that provide additional details about the application, device, user session, or environment.
+
+Set the following Global Attributes to instrument your application:
+
+```java Java
+Middleware.builder()
+        .setGlobalAttributes(
+            Attributes.builder()
+                    .put("key", "value")
+                    .put(StandardAttributes.APP_VERSION, BuildConfig.VERSION_NAME)
+                    .build());
+```
+### Custom Events [Optional]
+
+<AccordionGroup>   
+    <Accordion title="Send Custom Events">
+      Send custom events and workflows using `addEvent` 
+        ```java Java
+        Middleware.getInstance().addEvent("You clicked on Button", BUTTON_ATTRIBUES);
+        ```
+    </Accordion>
+
+    <Accordion title="Start Custom Event Workflow">
+      Start custom events and workflows using `startWorkflow`  
+        ```java Java
+        Span loginWorkflow = Middleware.getInstance().startWorkflow("User Login Flow");
+        ```
+    </Accordion>
+
+    <Accordion title="End Custom Event Workflow">
+      End custom events and workflows using `end`
+        ```java Java
+        loginWorkflow.end();
+        ```
+    </Accordion>
+</AccordionGroup>
+
+### Error Reporting [Optional]
+
+<Accordion title="Configure Error Reporting">
+  Use `addException(Throwable)` to report exceptions, errors, and display messages on the Middleware dashboard
+  ```java Java
+  Middleware.getInstance().addException(new RuntimeException("Something went wrong!"), Attributes.empty())
+  ```
+</Accordion>
+
+### Custom Logs [Optional]
+<Accordion title="Create Custom Logs">
+  Add custom logs to display on your Middleware dashboard such as debug, error, warning, and info 
+  ```java Java
+  Middleware logInstance = Middleware.getInstance();
+  logInstance.d("TAG", "I am debug");
+  logInstance.e("TAG", "I am error");
+  logInstance.i("TAG", "I am info");
+  logInstance.w("TAG", "I am warning");
+  ```
+</Accordion>
+
+# Attributes
```

Comment:
How does this look as not an accordian? I think the two downsides of accoridans are 1: things aren't searchable (well...they are, but Google won't actually move your index to them) and 2: the right nav bar doesn't have clickable options.

B/c the attributes are something that a user may be looking for when they come to this doc it's best to have them more exposed/accessible.

Reply:
I agree with both points. I think the big issue for me when I was placing them in accordions was on the accessibility front - I like to search variable names with command F in the documentation I use. 

That said, when I first placed all of these tables into the doc it looked terrible. It was a sort of long doom scroll to the bottom that was hard to track because of how much information there was. I prefer the dropdown boxes when they are in this document (for readability).  However, I can place them in a separate document or put them all in one big dropdown so that when a user clicks into the dropdown they can still command F the specific attribute they are looking for.

Reply:
@Zamichel

Reply:
Can we put each of the attribute headers as a sub header to attributes, with an accordian to expand them? That way the document nav on the right side will show all the different types of attributes, "Default/Resource/Instrumentation". I don't think a separate doc makes sense either.